### PR TITLE
 Fix EEFNameSpacedPropertyEditorDialog issues in Mediators

### PR DESCRIPTION
**Pull request by dilee**
## Purpose
- FilterMediator
- ForEachMediator
- HeaderMediator
- ScriptMediator

### CODE REVIEW

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/FilterMediatorPropertiesEditionPartForm.java [hunk 1]
```diff
@@ -783,6 +782,18 @@ public void focusGained(FocusEvent e) {
         sourceElements = new Control[] { itemLabel, sourceText, itemHelp };
         return parent;
     }
+    
+    private void openNamespacedPropertyEditor(final Composite parent) {
+        EEFNameSpacedPropertyEditorDialog nspd = new EEFNameSpacedPropertyEditorDialog(parent.getShell(),
+                SWT.NULL, source);
+        // valueExpression.setPropertyValue(valueExpressionText.getText());
```
Comment (prabushi) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/FilterMediatorPropertiesEditionPartForm.java]: Remove commented code.

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/ScriptMediatorPropertiesEditionPartForm.java [hunk 2]
```diff
@@ -1251,6 +1250,18 @@ public void focusGained(FocusEvent e) {
 		return parent;
 	}
 	
+    private void openNamespacedPropertyEditor(final Composite parent) {
+        EEFNameSpacedPropertyEditorDialog nspd = new EEFNameSpacedPropertyEditorDialog(parent.getShell(),
+                SWT.NULL, scriptDynamicKey);
+        // valueExpression.setPropertyValue(valueExpressionText.getText());
```
Comment (prabushi) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/ScriptMediatorPropertiesEditionPartForm.java]: Remove commented line.

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/FilterMediatorPropertiesEditionPartForm.java (hunk 1 above)
Comment (dilee) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/FilterMediatorPropertiesEditionPartForm.java]: Fixed in https://github.com/wso2/devstudio-tooling-esb/commit/1d53e06b88e566a6d316666c06dd48202d396d30

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/ScriptMediatorPropertiesEditionPartForm.java (hunk 2 above)
Comment (dilee) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/ScriptMediatorPropertiesEditionPartForm.java]: Fixed in 1d53e06

**Pull request merged**